src/components/Dialog/Dialog.jsx

```diff
@@ -346,12 +348,19 @@ export default class Dialog extends PureComponent {
               ...modifiers
             ]}
           >
-            {({ placement, style, ref, arrowProps }) => {
+            {({ placement, style, ref, arrowProps, isReferenceHidden }) => {
```

Comment:
@orrgottlieb are we ok with sending new objects/arrays every time to the popper component?

Reply:
it is the render props methodology, it is a render function. isReferenceHidden is a boolean @MosheZemah

Reply:
I meant the code above that line (that wasn't changed)